guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -63,13 +65,14 @@ For more information about the Discovery service, xref:configuring-the-discovery
 * A bare metal host or a blank VM.
 include::../common/modules/snip_common-compute-resource-prereqs.adoc[]
 
+ifndef::satellite[]
 For information about the security token for unattended and PXE-less provisioning, see xref:Provisioning_Bare_Metal_Hosts-Security_Token_in_the_Boot_Process[].
+endif::[]
 
 [[Provisioning_Bare_Metal_Hosts-Security_Token_in_the_Boot_Process]]
 === Configuring the Security Token Validity Duration
-
```

Comment:
Please don't remove this empty line below the header.

Also, this would mean there's an empty header in Satellite docs.